translate/src/modules/entitieslist/components/EntitiesListToggle.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import './EntitiesListToggle.css';
+
+/**
+ * Render entities list toggle.
+ *
+ * Used in a single-column layout.
+ */
+export function EntitiesListToggle(): React.ReactElement<'div'> {
+  const toggleEntitiesList = () => {
+    document
+      .querySelector('#app > .main-content')
+      .classList.toggle('entities-list');
```

Comment:
This is just a temporary hack to show the functionality. Will replace with proper state management before merging.